Makefile

```diff
@@ -9,7 +9,7 @@
 #
 # CGM_BASE_DIR = /path/to/installed/cgm
 
-include ${CGM_BASE_DIR}/lib/iGeom-Defs.inc
+-include ${CGM_BASE_DIR}/lib/iGeom-Defs.inc
```

Comment:
I believe it fails gracefully if the include fails. In the absence of the variable it will attempt to include `/lib/iGeom-Defs.inc` and continue. [GNU make include docs](https://www.gnu.org/software/make/manual/html_node/Include.html)